storage/id_alloc_test.go

```diff
@@ -176,7 +176,7 @@ func TestAllocateErrorAndRecovery(t *testing.T) {
 		}()
 	}
 	// Make sure no allocation returns.
-	time.Sleep(10 * time.Millisecond)
+	time.Sleep(250 * time.Millisecond)
```

Comment:
Why should this be necessary? Waiting a shorter amount of time shouldn't trouble a slow machine. This just adds a quarter second to every test run--we should resist making such changes.